### PR TITLE
Add bg zoom in effect on scroll

### LINKED ISSUE #1100: Animation effect on background image
### What would you like to share?

Currently, their is no effect on the background image. I want to give a zoom effect on scroll which will look really amazing on it.

### Additional information

_No response_

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR closes #1100

### CODE REVIEW

**Review by narayan954 — CHANGES_REQUESTED** [review]
hi @Palakkgoyal this looks awesome but it's not getting reset to original size after we scroll back up, can you please check?
image

image

**Comment by Palakkgoyal** [comment]
I did the required changes. This was occuring due to zoom out effect as in the useEffect I had given:
160 - pageYOffset
due to which the initial width and height of the image become 160%.
To solve this I changed it to zoom in instead of zoom out. It works fine now.

File: src/pages/Profile/index.jsx [hunk 1]
```diff
@@ -125,6 +127,18 @@ function Profile() {
     }
   };
 
+  useEffect(() => {
+    const bg = bgRef.current;
+    function handleScroll() {
+      bg.style.height = 100 + +window.pageYOffset/16 + "%";
```
Comment (narayan954) [inline src/pages/Profile/index.jsx]: pageyoffset is deprecated ,changing to scrollY instead

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**